src/pages/dmt5/_flexibility.tsx

```diff
@@ -169,7 +171,13 @@ const Flexibility = () => {
                                     </StyledHeader>
                                     {item.icon}
                                 </Flex>
-                                <StyledText>{item.text}</StyledText>
+                                {is_eu_country && is_uk ? (
```

Comment:
We can use `is_uk_eu` to cover both of the condition. `is_eu_country` is quite old and we are trying to replace them as much as we can